src/lib/types.ts

```diff
@@ -54,10 +49,29 @@ export interface IExpert {
     id: number;
     name: string;
   };
-  status?: ExpertStatus;
-  direction?: DirectionEnum;
-  email?: string;
-  phone?: string;
-  workPlace?: string;
-  lastPost?: string;
+  mainDirection?: {
+    id?: number;
+    color: string;
+    name: string;
+    route?: string;
+  };
```

Comment:
is this a repeated structure? 
can we get the direction for the post with these fields?
if so, we need a separate reusable interface for this structure, since it can be shared.

Reply:
created IDirectionProperty